packages/analytics-core/src/config.ts

```diff
@@ -42,6 +42,7 @@ export class Config implements IConfig {
   minIdLength?: number;
   plugins: Plugin[];
   optOut: boolean;
+  partnerId?: string;
```

Comment:
Carry over from previous RP: https://github.com/amplitude/Amplitude-TypeScript/pull/92#discussion_r872896534

Reply:
I might misunderstand the comment. So don't need `partnerId`, right?

Reply:
I believe so. Let's check the python SDK or have @bohan-amplitude confirm

Reply:
Removed. @bohan-amplitude could you please review and confirm it?

Reply:
Cool, thanks for the explanation. Resolve this conversation.